src/MassTransit.QuartzIntegration.Tests/Recurring_Specs.cs

```diff
@@ -104,6 +104,8 @@ public MySchedule()
 
                 StartTime = DateTime.Now + TimeSpan.FromSeconds(3);
                 EndTime = StartTime + TimeSpan.FromSeconds(7);
+
+                Description = "my description";

```

Comment:
Yes, this description is then will be writen to the DB. I think there can be scenarios (well, I have one) where client want to get all the schedules with meaningfull description and not only GUIDs. So in case we are using DB as job store we could query DB for jobs, triggers and trigger description.
About how to test... We need to test that this description is correctly mapped to ITrigger.Description. The way we can do this is to provide the ability to get an access to the scheduler instance (now this instance is created inside extension UseInMemoryScheduler() vie the new keyword). We can introduce some kind of static factory like so
```c#
public static class SchedulerFactory
{
    private readonly Func<IScheduler> _defaultFactory = () =>
    {
            ISchedulerFactory schedulerFactory = new StdSchedulerFactory();
            return schedulerFactory.GetScheduler();
    };

    private Func<IScheduler> _factory = _defaultFactory;;

    public static void SetFactory(Func<IScheduler> factory)
    {
        _factory = factory;
    }

    public static void SetDefault()
    {
        _factory = _defaultFactory;
    }

    public static IScheduler CreateScheduler() => _factory();
}
```
and refac extension method
```c#
    public static class QuartzIntegrationExtensions
    {
        public static void UseInMemoryScheduler(this IBusFactoryConfigurator configurator)
        {
            if (configurator == null)
                throw new ArgumentNullException(nameof(configurator));

            var scheduler = SchedulerFactory.CreateScheduler();

            var specification = new InMemorySchedulerBusFactorySpecification(scheduler);
            configurator.AddBusFactorySpecification(specification);

            ...
        }
    }
```
Having this we can create a Mock<IScheduler> in tests and provide this mock to the factory. After that we only need to verify, that IScheduler.ScheduleJob(IJobdetails, ITrigger) was called with expected params (check ITrigger.Description property).